src/services/factories/tokenList.ts

```diff
@@ -218,8 +258,20 @@ export function getTokensFactory(factoryParams: {
     // Get token details for each filtered token
     const tokenDetails = await _fetchTokenDetails(networkId, filteredAddressesAndIds, tokensConfig)
 
+    // Sort tokens
+    const tokenList = _sortTokens(networkId, tokenDetails)
+
     // Persist it
-    tokenListApi.persistTokens({ networkId, tokenList: tokenDetails })
+    tokenListApi.persistTokens({ networkId, tokenList })
+  }
+
+  function _tokenComparer(a: TokenDetails, b: TokenDetails): number {
+    if (a.label < b.label) {
+      return -1
+    } else if (a.label > b.label) {
+      return 1
+    }
+    return 0
```

Comment:
considering what a lable may be (unicode string)and for simpicity, I recommend `String.localeCompare` (or `Intl.collator.compare` if you fancy that)